### PR TITLE
Fix message on signup

### PR DESCRIPTION
**Pull request by tsauerwein**
Closes https://github.com/c2corg/v6_ui/issues/592

### CODE REVIEW

File: c2corg_ui/static/js/auth/auth.js [hunk 1]
```diff
@@ -226,7 +226,9 @@ app.AuthController.prototype.register = function() {
   form['lang'] = lang; // inject the current language
 
   this.api_.register(form).then(function() {
-    var msg = alerts.gettext('Register success');
+    var msg = alerts.gettext(
+        'Thank you for your registration! You will receive a confirmation' +
+        'mail in an instance.');
```
Comment (asaunier) [inline c2corg_ui/static/js/auth/auth.js]: In an instance?
What about simply "you will soon receive a confirmation email" (or "validation"?)?

Comment (tsauerwein) [inline c2corg_ui/static/js/auth/auth.js]: Yes, that sounds better. :+1:

**Comment by asaunier** [comment]
@tsauerwein I have pushed a fixup commit to improve the message. If it's OK for you I will squash the commits and merge the PR.

**Comment by tsauerwein** [comment]
Thanks, please merge.

**Pull request merged**